extensions/visualizations/visualizations.js

```diff
@@ -47,9 +47,7 @@ oppia.directive('oppiaVisualizationBarChart', [function() {
             title: 'Number of times answer was submitted',
             minValue: 0
           },
-          legend: {
-            position: 'none'
-          },
+          legend: null,
```

Comment:
Why this change?

Reply:
Legend in options is typed as follows:

![image](https://user-images.githubusercontent.com/15226041/54277221-46c19680-45b5-11e9-8bb6-e534af7884f6.png)

ChartLegend is typed as follows:

![image](https://user-images.githubusercontent.com/15226041/54277276-6d7fcd00-45b5-11e9-81f8-490228f014dd.png)

So, if we want legend to be none, we cannot specify position as none (that is not an allowed type for position), instead, we directly need to specify legend as none.

Reply:
That's odd -- the examples on https://developers.google.com/chart/interactive/docs/gallery/barchart, we have a legend dict. Is it possible that that legend dict should be changed to a ChartLegend object instead?

That said, I'm fine with null anyway. It seems like what was intended in the first place.